### DISCUSSION TITLE
Would it be possible to use Privacy Badger in sorting search results based on the amount of trackers?

### DISCUSSION THREAD

**Discussion by arvidmarklund** [EFForg/privacybadger#2815] [Ideas]
Hi, would it be possible to utilize the privacy badger learning in sorting search results (to stop tracking intensive sites to reach the top of the list). Are there any plans for something like that as an extension/fork of privacy badger? Maybe like a meta-search with anti-tracking and/or anti-bloat focus. I would be very interested in such a project.

**Comment by ghostwords** [comment]
Hello!

Sure, you could look into using [Badger Sett](https://github.com/EFForg/badger-sett) to figure out which sites have the most trackers. There are tricky bits there; please see the discussion in #2716 for more information. It would be easier though to start with an existing data set, perhaps https://whotracks.me/ or https://spreadprivacy.com/duckduckgo-tracker-radar/.

Once you gather a list of tracking heavy sites, you could then look for or make an extension to automatically filter out those sites from search engine results.

**Reply by arvidmarklund** [reply]
Thanks! I will definitely look into that!
